frontend/src/hooks/api/useMarkAsPlayed.ts

```diff
@@ -5,8 +5,8 @@ export const useMarkAsPlayed = () => {
 	const queryClient = useQueryClient();
 
 	return useMutation({
-		mutationFn: (data: { gameId: string }) =>
-			axios.post('/api/v1/games/markPlayed', data),
+		mutationFn: (data: { gameId: string, played: boolean }) =>
+			axios.post(`/api/v1/games/${data.played ? 'markNotPlayed' : 'markPlayed'}/${data.gameId}`),
```

Comment:
You should probably rename the function to something like "useChangePlayStatus" to reflect how it actually works now.